doc/releases/changelog-0.21.0.md

```diff
@@ -120,125 +217,20 @@
-  0: ──╭C──RY(0.1)───────────────────────────────┤
-  1: ──╰X──RY(-0.3)──╭C──RY(0.4)─────────────────┤
-  2: ────────────────╰X──RY(0.2)──╭C──RY(-0.15)──┤
-  3: ─────────────────────────────╰X──RY(0.5)────┤ ⟨Z⟩
-  ```
-* Added a template for tree tensor networks (TTN).
-  [(#2043)](https://github.com/PennyLaneAI/pennylane/pull/2043)
-  An example circuit that uses the `TTN` template is:
-  ```python
-  import pennylane as qml
-  import numpy as np
-
-  def block(weights, wires):
-      qml.CNOT(wires=[wires[0],wires[1]])
-      qml.RY(weights[0], wires=wires[0])
-      qml.RY(weights[1], wires=wires[1])
-
-  n_wires = 4
-  n_block_wires = 2
-  n_params_block = 2
-  n_blocks = qml.MPS.get_n_blocks(range(n_wires),n_block_wires)
-  template_weights = [[0.1,-0.3]]*n_blocks
-
-  dev= qml.device('default.qubit',wires=range(n_wires))
-  @qml.qnode(dev)
-  def circuit(template_weights):
-      qml.TTN(range(n_wires), n_block_wires, block, n_params_block, template_weights)
-      return qml.expval(qml.PauliZ(wires=n_wires-1))
-  ```
-  The resulting circuit is:
-  ```pycon
-  >>> print(qml.draw(circuit,expansion_strategy='device')(template_weights))
-  0: ──╭C──RY(0.1)─────────────────┤
-  1: ──╰X──RY(-0.3)──╭C──RY(0.1)───┤
-  2: ──╭C──RY(0.1)───│─────────────┤
-  3: ──╰X──RY(-0.3)──╰X──RY(-0.3)──┤ ⟨Z⟩
-  ```
-
-* Functions for tapering qubits based on molecular symmetries is added.
-  [(#1966)](https://github.com/PennyLaneAI/pennylane/pull/1966)
-  [(#1974)](https://github.com/PennyLaneAI/pennylane/pull/1974)
-  [(#2041)](https://github.com/PennyLaneAI/pennylane/pull/2041)
-
-  With this functionality, a molecular Hamiltonian and the corresponding Hartree-Fock (HF) state can be transformed to a new Hamiltonian and HF state that acts on a reduced number of qubits, respectively.
-
-  ```python
-  from pennylane import hf
-  from pennylane import numpy as np
-
-  symbols = ["He", "H"]
-  geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.4588684632]])
-  mol = hf.Molecule(symbols, geometry, charge=1)
-  H = hf.generate_hamiltonian(mol)(geometry)
-  n_qubits, n_elec = len(H.wires), mol.n_electrons
-
-  generators, paulix_ops = hf.generate_symmetries(H, n_qubits)
-  opt_sector = hf.optimal_sector(H, generators, n_elec)
-  H_tapered = hf.transform_hamiltonian(H, generators, paulix_ops, opt_sector)
-  hf_tapered = hf.transform_hf(generators, paulix_ops, paulix_sector,
-                                    n_elec, n_qubits)
-  ```
-  ```pycon
-  >>> print(H_tapered)
-    ((-1.7997297644914574+0j)) [I0]
-  + ((-0.10492941956079854+0j)) [X0]
-  + ((0.10492941956079856+0j)) [X1]
-  + ((0.5675134088336165+0j)) [Z1]
-  + ((0.5675134088336168+0j)) [Z0]
-  + ((-0.14563730440190722+0j)) [Y0 Y1]
-  + ((-0.10492941933657857+0j)) [X0 Z1]
-  + ((0.09337410512815508+0j)) [Z0 Z1]
-  + ((0.10492941933657857+0j)) [Z0 X1]
-  >>> print(hf_tapered)
-  tensor([1, 1], requires_grad=True)
-  ```
-
+<h4>Adjoint metric tensor</h4>
```

Comment:
```suggestion
<h4>Speedier quantum natural gradient</h4>
```
Because I think more people are familiar with QNG than metric tensor :)

Reply:
Alternatively, we could make this 
```suggestion
<h4>More efficient quantum gradient transforms ⚡</h4>
```
to also include the Hessian entry, what do you think? Not _fully_ accurate --- since the metric tensor is not a gradient --- but maybe 'close enough'?

Reply:
Yep, I was also contemplating about having these two together :thinking: I think that the separate headlines that were suggested could catch the attention of the users better. I'll apply those (unless we'd like to merge the two entries surely).